### PR TITLE
docs: adding field descriptions to predefined mesh 3D document

### PR DESCRIPTION
**Pull request by punndcoder28**
* Add Field Description and examples to predefined `Mesh3D` document #1736

### CODE REVIEW

**Comment by punndcoder28** [comment]
@JoanFM This PR addresses the changes to add field descriptions and examples for `Mesh3D` document

File: docarray/documents/mesh/mesh_3d.py [hunk 1]
```diff
@@ -103,10 +106,24 @@ class MultiModalDoc(BaseDoc):
...
+        default=None,
+    )
+    tensors: Optional[VerticesAndFaces] = Field(
+        description='A tensor object of 3D mesh of type `VerticesAndFaces`.',
+        example=[[0, 1, 1], [1, 0, 1], [1, 1, 0]],
+        default=None,
+    )
+    embedding: Optional[AnyEmbedding] = Field(
+        description='Store an embedding: a vector representation of the 3D mesh.',
+        default=[[1, 0, 1], [0, 1, 1], [1, 1, 0]],
```
Comment (JoanFM) [inline docarray/documents/mesh/mesh_3d.py:121]: the embedding is tipically a 1-D vector, so better show that in example

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: docarray/documents/mesh/mesh_3d.py (hunk 1 above)
Comment (punndcoder28) [reply docarray/documents/mesh/mesh_3d.py:121]: Understood but the docs for this document says the below
```
Vertices are points in a 3D space, represented as a tensor of shape (n_points, 3)
```

Comment (JoanFM) [reply docarray/documents/mesh/mesh_3d.py]: buut vertices are not the embedding. The embedding is a vector semantically representing an object, in this case a 3D mesh represented by its vertices.

**Review by JoanFM — APPROVED** [review]

**Pull request merged**